views/sms-settings.ejs

```diff
@@ -90,6 +90,32 @@
         <select id="bday-year" name="bday-year"></select>
       </div>
     </fieldset>
+    
+    <fieldset>
+      <div class="label-container">
+        <label>What's one thing you want to work on most?</label>
+        <p>Focusing on ONE area is key.</p>
+      </div>
+      <div class="input-container">
+        <select id="intention-category" name="intention-category">
+          <option value="presence" selected="selected">Joy & Presence</option>
```

Comment:
these option values are currently set to match the string values extracted in lightbeam logic:
https://github.com/shinetext/lightbeam/blob/dev/tasks/mc-profiles.js#L190